static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.tsx

```diff
@@ -0,0 +1,300 @@
+import React, {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+import uniqBy from 'lodash/uniqBy';
+
+import Alert from 'sentry/components/alert';
+import {Button} from 'sentry/components/button';
+import ExternalLink from 'sentry/components/links/externalLink';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import {IconWarning} from 'sentry/icons';
+import {t, tct, tn} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import type {PlatformType} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {
+  SourceMapDebugError,
+  SourceMapDebugResponse,
+  SourceMapProcessingIssueType,
+  StacktraceFilenameQuery,
+  useSourceMapDebugQueries,
+} from './useSourceMapDebug';
+
+const platformDocsMap = {
+  'javascript-react': 'react',
+  'javascript-angular': 'angular',
+  'javascript-angularjs': 'angularjs',
+  'javascript-backbone': 'backbone',
+  'javascript-ember': 'ember',
+  'javascript-gatsby': 'gatsby',
+  'javascript-vue': 'vue',
+  'javascript-nextjs': 'nextjs',
+  'javascript-remix': 'remix',
+  'javascript-svelte': 'svelte',
+};
+
+const shortPath = ['javascript', 'node'];
+
+function getErrorMessage(
+  error: SourceMapDebugError,
+  platform: PlatformType
+): Array<{
+  title: string;
+  /**
+   * Expandable description
+   */
+  desc?: string;
+  docsLink?: string;
+}> {
+  switch (error.type) {
+    case SourceMapProcessingIssueType.MISSING_RELEASE:
+      return [
+        {
+          title: tct('Update your [init] call to pass in the release argument', {
+            init: <code>Sentry.init</code>,
+          }),
+          docsLink: shortPath.includes(platform)
+            ? `https://docs.sentry.io/platforms/${platform}/configuration/options/#release`
+            : `https://docs.sentry.io/platforms/javascript/guides/${platformDocsMap[platform]}/configuration/options/#release`,
+        },
+        {
+          title: t(
+            'Integrate Sentry into your release pipeline. You can do this with a tool like webpack or using the CLI. Not the release must be the same as in step 1.'
+          ),
+          docsLink: shortPath.includes(platform)
+            ? `https://docs.sentry.io/platforms/${platform}/sourcemaps/#uploading-source-maps-to-sentry`
+            : `https://docs.sentry.io/platforms/javascript/guides/${platformDocsMap[platform]}/sourcemaps/#uploading-source-maps-to-sentry`,
```

Comment:
this link is invalid for `angularjs` and `backbone` as well. I haven't checked the others but it's probably the same situation